android/modules/platform/src/java/ti/modules/titanium/platform/DisplayCapsProxy.java

```diff
@@ -67,10 +67,16 @@ public String getDensity() {
 				return "high";
 			case DisplayMetrics.DENSITY_MEDIUM :
 				return "medium";
-			case 320 : // DisplayMetrics.DENSITY_XHIGH (API 9)
+			case DisplayMetrics.DENSITY_XHIGH :
```

Comment:
XHIGH should be fine since min support is 10 now. I'll add the TV and switch them to numbers.